R/constants.R

```diff
@@ -0,0 +1,33 @@
+## ----
+# Specified BrAPI endpoints
+BRAPI_ENDPOINTS <- list(
+    "METHOD_TABLE"   = "allelematrix",
+    "SAMPLES"        = "samples",
+    "SERVER_INFO"    = "serverinfo",
+    "VARIANT_TABLES" = "variantTables",
```

Comment:
Should variantTables have a capital T?

Reply:
This is something that was transferred over from the v1 ktor service. I will keep it in for now until the v2 service matures.